x-pack/metricbeat/mbreceiver/receiver_test.go

```diff
@@ -237,3 +243,51 @@ func getFromSocket(t *testing.T, sb *strings.Builder, socketPath string) bool {
 	}
 	return true
 }
+
+func BenchmarkFactory(b *testing.B) {
```

Comment:
Am I wrong or it isn't used anywhere?
Perhaps you thought about using it [here](https://github.com/elastic/beats/pull/44127/files#diff-10aef7bb19997b6f9b2fa49fd8aee8c795571b57c0e14a486ac8b89543ad9794R151) but forgot to replace the original call?

Reply:
This isn't used anywhere. I don't think we consistently run benchmarks in CI. It's mainly for local use, to identify bottlenecks or potential improvements.